frog/imports/client/App/index.js

```diff
@@ -201,6 +201,20 @@ const FROGRouter = withRouter(
       });
       if (!this.wait) {
         const query = queryToObject(this.props.location.search.slice(1));
+        if (query.setAdmin) {
+          LocalSettings.UrlCoda = '?setAdmin=' + query.setAdmin;
```

Comment:
You shouldn't need the UrlCoda, it's only needed for things we want to be attached to the URL everytime you change routing.